frontend/src/components/Teams/TeamsList/TeamItem/index.tsx

```diff
@@ -30,16 +30,16 @@ const InnerContainer = styled(Flex, Box, {
 });
 
 export type TeamItemProps = {
-  userId: string | undefined;
   team: Team;
   isTeamPage?: boolean;
 };
 
-const TeamItem = React.memo<TeamItemProps>(({ userId, team, isTeamPage }) => {
+const TeamItem = React.memo<TeamItemProps>(({ team, isTeamPage }) => {
   const { data: session } = useSession();
 
   const router = useRouter();
 
+  const userId = session?.user.id;
   const isSAdmin = session?.user.isSAdmin;
```

Comment:
```suggestion
    const { userId = undefined, isSAdmin = undefined } = { ...session };
```

This is just a suggestion on how we can destructuring an undefined object.
https://stackoverflow.com/questions/48433008/js-es6-destructuring-of-undefined